pydarn/plotting/rtp.py

```diff
@@ -920,10 +938,11 @@ def plot_summary(cls, dmap_data: List[dict], beam_num: int = 0,
                                         ax=axes[i],
                                         groundscatter=grndflg,
                                         channel=channel,
+                                        slant=slant,
                                         cmap=cmap[axes_parameters[i]],
                                         zmin=boundary_ranges[axes_parameters[i]][0],
                                         zmax=boundary_ranges[axes_parameters[i]][1],
-                                        ymax=75,
+                                        ymax=3517.5,
```

Comment:
I guess this might be where you need an if statement, re: the comment from earlier?

Reply:
yup fixed